### ISSUE TITLE
Re-establish dynamic loading of KORP frontend config files

### ISSUE THREAD

**Issue by simongray** [kuhumcst/infrastructure#7]
It used to be the case in the [old KORP installation](https://alf.hum.ku.dk/korp-old/) that people - e.g. Dorte - could manually edit the frontend config files, have it reload automatically, and use that as a quick feedback mechanism when developing a new KORP corpora setup. While this is not exactly a desirable property to have in a live production system, this is her desired workflow and we should work to support it. In the future, when she has a fully local installation on her computer, she (and others) will be happy to have this functionality there as well.

I speculate that the reason that [the current KORP installation](https://alf.hum.ku.dk/korp/) doesn't support is has to do with the fact that I ignore the `run_config.json` file entirely, preferring to simply patch the KORP frontend file structure directly as that provided me with patch access to the entirety of the KORP frontend codebase, as opposed to just the files allowed through a `run_config.json` file. A hybrid solution where patching happens, but the `run_config.json` also exists might be deliver the best of both worlds.

**Comment by simongray** [comment]
After spending more than a full day's work on trying to figure out `run_config.json` I have discovered that it basically offers nothing over the existing copy-in-place approach. It is run exactly once - during compilation - so the naming really makes no sense. I am beginning to suspect that someone at Språkbanken thinks runtime is the same as compile time. In a JS setting, runtime would be when the app is run in the browser, not when the JS bundle is being compile.

Anyway, I discovered that the `dist/` folder created internally in the Docker container during the compilation step contains a `translations` and a `modes` directory (as well as `markup` and `lib` which don't care about). Files can be dynamically reloaded by copying them into these folders. In order to allow Dorte to change mode I should put a script inside the docker container that she can run using `docker-compose exec` and this script should simply copy `app/translations` and `app/modes` into `dist` in order to sync the changes.
